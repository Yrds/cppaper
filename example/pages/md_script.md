{{ lua("return string.upper('thanks to lua super powers I became uppercased')") }}
