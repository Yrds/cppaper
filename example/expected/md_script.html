<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width" />
    <title>Yuri Santos - md_script</title>
  </head>
  <body>
    <p>THANKS TO LUA SUPER POWERS I BECAME UPPERCASED</p>

  </body>
</html>
